lib/src/builders/sticker.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:nyxx/src/builders/builder.dart';
+import 'package:nyxx/src/builders/image.dart';
+import 'package:nyxx/src/models/sticker/guild_sticker.dart';
+
+class StickerCreateBuilder implements CreateBuilder<GuildSticker> {
+  /// Name of the sticker (2-30 characters)
+  final String name;
+
+  /// Description of the sticker (empty or 2-100 characters)
+  final String? description;
+
+  /// Autocomplete/suggestion tags for the sticker (max 200 characters)
+  final String tags;
+
+  /// The sticker file to upload
+  final ImageBuilder file;
+
+  StickerCreateBuilder({required this.name, required this.description, required this.tags, required this.file});
```

Comment:
```suggestion
  StickerCreateBuilder({required this.name, this.description = '', required this.tags, required this.file});
```
Also change `description` from `String?` to `String`.

Since all we do if description is `null` is pass an empty string, might as well make it the default value and never allow null. We also don't need it to be required.